src/executables/csql_result.c

```diff
@@ -485,6 +485,8 @@ display_empty_result (int stmt_type, int line_no)
   return;
 }
 
+#define IS_STRING_TYPE(type)     (type == DB_TYPE_STRING || \
+                                  type == DB_TYPE_CHAR)
```

Comment:
Is there a reason to define it separately rather than using PT_IS_STRING_TYPE()?
I am concerned that I am missing processing for the VARCHAR type.

Reply:
Good point. I fixed to to use TP_IS_CHAR_TYPE defined at "object_domain.h".